### ISSUE TITLE
Problem when uninstalling!

### ISSUE THREAD

**Issue by PerPalmgren** [jasp-stats/jasp-issues#1440]
* JASP version: 0.15

* OS name and version: Windows
* Analysis: I ma very happy to see the new JASP version. You have done a great job guys :-). I love your program. I am using the program for our doctoral students but the IT-department says there is a problem uninstalling JASP, they mean there is an error in your msi-file, thus when it is going to uninstall it is trying to install once more. Therefore our IT-department cant post the JASP program in our Software Centre. Is this something you are working on? Kind regards Per
* Bug description: 
* Expected behaviour:

**Comment by boutinb** [comment]
Yes we have seen this error. Cf https://github.com/jasp-stats/jasp-issues/issues/1438
Maybe they can have a look to this version https://static.jasp-stats.org/Nightlies/Windows/JASP-nightly-development-580450fc96173b0ea077bebe8f7e8b0d91998fd5.msi (this is not a signed version, so Windows will warn you not to use it).
If this fix solves your problem we can maybe add this fix as a hotfix for the 0.15 release.

**Comment by boutinb** [comment]
I have made a new signed msi file for 0.15. If you now download and install the JASP 0.15 version for Windows, you should get the right uninstall command in the registry.
